translator/internaldata/oc_to_metrics_test.go

```diff
@@ -433,15 +433,18 @@ func TestOCToOTLP(t *testing.T) {
 		},
 	}
 
+	emptyMetricData := data.NewMetricData()
+	emptyMetricData.ResourceMetrics().Resize(1)
+
 	tests := []struct {
-		name string
-		oc   consumerdata.MetricsData
-		otlp []*otlpmetrics.ResourceMetrics
+		name     string
+		oc       consumerdata.MetricsData
+		internal data.MetricData
 	}{
 		{
-			name: "empty",
-			oc:   consumerdata.MetricsData{},
-			otlp: nil,
+			name:     "empty",
+			oc:       consumerdata.MetricsData{},
+			internal: data.NewMetricData(),
 		},
```

Comment:
Suggest to use tests from testdata.

Reply:
done. I've reused some existing OC test metrics